src/OtpInterface.php

```diff
@@ -47,10 +47,11 @@ function totp($secret, $timecounter = null);
      * @param string  $secret  Base32 Secret String
      * @param integer $counter Counter
      * @param string  $key     User supplied key
+     * @param integer $counterwindow Size of the look-ahead window
      *
-     * @return boolean True if key is correct
+     * @return int|boolean the counter if key is correct else false
      */
-    function checkHotp($secret, $counter, $key);
+    function checkHotp($secret, $counter, $key, $counterwindow);
```

Comment:
More `unction checkHotp($secret, $counter, $key, $counterwindow = 2);` otherwise you dfine that counterwindow MUST be present and that would be a BC break. But thankfully that is not needed, so you can just kill that bit :)